libs/ui/src/components/LibraryItemsList/MenuItemList/MenuItemList.test.tsx

```diff
@@ -72,4 +72,31 @@ describe('MenuItemList', () => {
 
         expect(screen.getByText('DisplayOptions')).toBeInTheDocument();
     });
+
+    describe('refetch after creation', () => {
+        beforeEach(() => {
+            kitNotificationMock.success.mockClear();
+        });
+
+        it.each([LibraryBehavior.standard, LibraryBehavior.files, LibraryBehavior.directories])(
```

Comment:
on utilise `test` plutôt que `it` dans LEAV. Je n'ai rien contre `it` personnellement, simple question de cohérence